templates/docs/whats-new.md

```diff
@@ -11,6 +11,30 @@ context:
 When we add, make significant updates, or deprecate a component we update their status so that it’s clear what’s available to use. Check back here anytime to see current status information.
 
 <table aria-label="What's new in Vanilla {{versionMinor}}">
+  <thead>
+    <tr>
+      <th style="width: 20%">Component</th>
+      <th style="width: 15%">Status</th>
+      <th style="width: 10%">Version</th>
+      <th style="width: 55%">Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <!-- 3.3.0 -->
+    <tr>
+      <th><a href="/docs/patterns/navigation#expanding-search-box">Segmented control</a></th>
```

Comment:
Link is copied from navigation, should link to new segmented button page.

Reply:
Ops!